src/natcap/invest/carbon.py

```diff
@@ -138,16 +138,18 @@
         "discount_rate": {
             "type": "number",
             "required": "do_valuation",
-            "about": "The discount rate as a floating point percent.",
-            "name": "Market Discount in Price of Carbon (%)"
+            "about": (
+                "The discount rate in the price of carbon as a floating "
+                "point percent."),
+            "name": "market discount rate (r)"

```

Comment:
It's a variable as used in the user's guide. I put it there because "discount rate" and "annual rate of change" are kind of similar. But now I'm thinking I'll take it out. I wish there was a better way to express what each represents.

Reply:
I'm going with "market discount rate" and "carbon price change", I think that sums it up alright.